src/main/java/com/bmc/truesight/meter/plugin/jira/JiraPlugin.java

```diff
@@ -107,28 +107,32 @@ public void run() {
                 if (isTemplateParsingSuccessful) {
                     TemplateValidator templateValidator = new JiraTemplateValidator();
                     try {
+                        if (config.getApp_id() == null) {
+                        } else if (StringUtil.isValidValue(config.getApp_id())) {
```

Comment:
we should not be using blank body, instead we can put it in a different way with ! operator.